packages/es-dev-server/src/middleware/compatibility-middleware.js

```diff
@@ -0,0 +1,102 @@
+import stripAnsi from 'strip-ansi';
+import { DEFAULT_EXTENSIONS } from '@babel/core';
+import { getBodyAsString, getRequestFilePath, isPolyfill } from '../utils/utils.js';
+import { sendMessageToActiveBrowsers } from '../utils/message-channel.js';
```

Comment:
this changes are unrelated? remnants of a bad rebase/merge maybe?

Reply:
ugh

Reply:
fixed